packages/paste-core/utilities/media-object/src/index.tsx

```diff
@@ -1,50 +1,45 @@
-import styled from '@emotion/styled';
-import {space, Theme} from 'styled-system';
-import {themeGet} from '@styled-system/theme-get';
-import {SpaceProps} from '@twilio-paste/style-props';
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Space} from '@twilio-paste/style-props';
 
-export interface MediaObjectProps extends Theme {
-  isCentered?: boolean;
+export interface MediaObjectProps {
+  verticalAlign?: 'center' | 'top';
 }
 
-const MediaObject = styled.div<MediaObjectProps>`
-  align-items: ${(props: MediaObjectProps) => (props.isCentered ? 'center' : 'flex-start')};
-  display: flex;
-`;
+const MediaObject: React.FC<MediaObjectProps> = ({children, verticalAlign = 'top'}) => {
+  return (
+    <Box display="flex" alignItems={verticalAlign === 'top' ? 'flex-start' : 'center'}>
```

Comment:
Should we use the Flex component here? 🤔

Reply:
Thought about it, but there reason I didn't was that Alert would require Media-object as a peer, then that has a peer on Flex, and that has a peer on Box, and I use _so little_ of Flex it just didn't seem worth it